tests/markdown_table/table.md

```diff
@@ -0,0 +1,12 @@
+- min-table
+  | Age | Time | Food | Gold | Requirement |
```

Comment:
> The nested list table is now not executed in the prettier. Should I stay the same?

I guess you meant that the table is not correctly recognized, you need to add a newline here to get it work:

```suggestion

  | Age | Time | Food | Gold | Requirement |
```